docs/current/sdk/nodejs/snippets/.eslintrc.cjs

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "prettier",
+    "plugin:prettier/recommended",
+  ],
+  ignorePatterns: [
+    "dist/",
+
+    // FIXME: generated files should be linted
+    "api/client.gen.ts",
+    "api/types.ts",
```

Comment:
The line is not necessary. `types.ts` will be removed

Reply:
Done